README.md

```diff
@@ -1 +1,96 @@
-# fast-budget
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Fast Budget
+
+> Mobile web application where you can manage your budget.
+
+You have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## Live Server
+
+You can check out the website [here](https://fast-budget-rails.herokuapp.com/).
```

Comment:
- [ ] Your application is working as expected locally. But ran into an error when creating a user using the live application. It will be nice if you could please look at this and try to fix it.

Test Demo of the live app |
:---------------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/170302401-92d46293-85cd-4dc5-a651-00b8a60f4f56.gif)